compiler/rustc_resolve/src/diagnostics.rs

```diff
@@ -120,6 +122,20 @@ fn reduce_impl_span_to_impl_keyword(sm: &SourceMap, impl_span: Span) -> Span {
     sm.span_until_whitespace(impl_span)
 }
 
+struct AmbiguousErrorDiag {
```

Comment:
```suggestion
struct AmbiguityErrorDiag {
```
For naming consistency.